src/applications/find-forms/components/FormTitle.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const FormTitle = ({ id, formUrl, title }) => (
+const FormTitle = ({ id, formUrl, showFindFormsMVPEnhancements, title }) => (
   <dt
     className="vads-u-padding-top--3 vads-u-margin--0 vads-u-border-top--1px vads-u-border-color--gray-lighter vads-u-font-weight--bold"
     data-e2e-id="result-title"
   >
-    {formUrl ? (
+    {formUrl && showFindFormsMVPEnhancements ? (
```

Comment:
Small suggestion on the the term `showFindFormsMVPEnhancements` - I think it would be nice to name the feature flag with what the MVP enhancement is composed of so that we have an idea of the effects of that feature flag. Something like "showFindFormsResultsLinkToFormDetailPages` (very verbose) or something along those lines so that it's clear exactly what the feature flag is driving

Reply:
Ah great point! Thanks for the feedback!